Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -296,6 +297,7 @@ def bump_formula_pr
         ohai "hub pull-request --browse -m '#{formula.name} #{new_formula_version}#{devel_message}'"
         ohai "git checkout -"
       else
+        safe_system "git", "fetch", "--unshallow", "origin"
```

Comment:
Optionally, we chould conditionally perform `--unshallow` only if the git repository was shallow to begin with. However, since `brew update` ran just prior to this, this should return fast if there's nothing to do. Let me know what you think is best.